integration/kube_integration_test.go

```diff
@@ -85,16 +85,11 @@ type KubeSuite struct {
 }
 
 func newKubeSuite(t *testing.T) *KubeSuite {
-	testEnabled := os.Getenv(teleport.KubeRunTests)
```

Comment:
Should we keep ability to skip to tests can be run locally? And have GHA workflow just set the variable.